tests/src/test/scala/scalaz/ValidationTest.scala

```diff
@@ -77,12 +78,12 @@ object ValidationTest extends SpecLite {
     "return first success" in {
       val succ = Validation.success[String, Int](1).toValidationNel
       val fail = Validation.failure[String, Int]("2").toValidationNel
-    
```

Comment:
Or we should contribute more ;-)

All folks I have shown the hook instantly adopted it so I'm quite confident.

Maybe something to discuss on the mailing list?